<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EVE万能工具 By: Sir丶雨轩 高达</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/favicon.ico</normaloff>:/favicon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabMain">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <widget class="QWidget" name="tabLp">
       <attribute name="title">
        <string>忠诚点兑换</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QTreeView" name="lpTree">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0" rowspan="2">
               <widget class="QPushButton" name="pushButton_2">
                <property name="text">
                 <string>搜索</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTabWidget" name="tabWidget_2">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="searchTab">
              <attribute name="title">
               <string>搜索结果</string>
              </attribute>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QTreeWidget" name="treeWidget">
                 <column>
                  <property name="text">
                   <string notr="true">1</string>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="myTab">
              <attribute name="title">
               <string>我的收藏</string>
              </attribute>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QWidget" name="widget_3" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>此功能尚未开发</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabStar">
       <attribute name="title">
        <string>行星开发</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabStarPlan">
           <attribute name="title">
            <string>方案组合</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">color:blue;</string>
              </property>
              <property name="text">
               <string>可用的行星</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListView" name="listView"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabAbout">
       <attribute name="title">
        <string>关于我们</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:24pt; font-weight:600; color:#0055ff; vertical-align:sub;&quot;&gt;EVE万能查询工具&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:24pt; font-weight:600; color:#0055ff; vertical-align:sub;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:18pt; font-weight:600; color:#ff0000; vertical-align:super;&quot;&gt;开发人员：Sir丶雨轩，高达&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:18pt; font-weight:600; color:#ff0000; vertical-align:super;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:18pt; font-weight:600; color:#aa55ff; vertical-align:super;&quot;&gt;功能紧张刺激的开发中&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:18pt; font-weight:600; color:#ff0000; vertical-align:super;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="priceAction"/>
   <addaction name="lpAction"/>
   <addaction name="refineAction"/>
   <addaction name="blueprintAction"/>
   <addaction name="starAction"/>
   <addaction name="agentAction"/>
   <addaction name="goodsAction"/>
   <addaction name="womAction"/>
   <addaction name="actionTask"/>
   <addaction name="actionDed"/>
   <addaction name="actionArc"/>
   <addaction name="actionAbyss"/>
   <addaction name="actionEden"/>
   <addaction name="aboutAction"/>
  </widget>
  <action name="starAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/2267_64.png</normaloff>:/2267_64.png</iconset>
   </property>
   <property name="text">
    <string>行星开发</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="lpAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/57484_64.png</normaloff>:/57484_64.png</iconset>
   </property>
   <property name="text">
    <string>忠诚点兑换</string>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="priceAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/price.png</normaloff>:/price.png</iconset>
   </property>
   <property name="text">
    <string>价格中心</string>
   </property>
   <property name="toolTip">
    <string>价格中心</string>
   </property>
  </action>
  <action name="refineAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/tilian.png</normaloff>:/tilian.png</iconset>
   </property>
   <property name="text">
    <string>提炼与再生</string>
   </property>
  </action>
  <action name="blueprintAction">
   <property name="icon">
    <iconset>
     <normaloff>:/blueprint.png</normaloff>:/blueprint.png</iconset>
   </property>
   <property name="text">
    <string>蓝图计算</string>
   </property>
   <property name="toolTip">
    <string>蓝图计算</string>
   </property>
  </action>
  <action name="agentAction">
   <property name="icon">
    <iconset theme="代理人搜索">
     <normaloff>:/agent.png</normaloff>:/agent.png</iconset>
   </property>
   <property name="text">
    <string>代理人搜索</string>
   </property>
  </action>
  <action name="goodsAction">
   <property name="icon">
    <iconset>
     <normaloff>:/corp/1000042.png</normaloff>:/corp/1000042.png</iconset>
   </property>
   <property name="text">
    <string>物品数据库</string>
   </property>
  </action>
  <action name="womAction">
   <property name="icon">
    <iconset>
     <normaloff>:/corp/1000049.png</normaloff>:/corp/1000049.png</iconset>
   </property>
   <property name="text">
    <string>虫洞查询</string>
   </property>
  </action>
  <action name="actionTask">
   <property name="icon">
    <iconset>
     <normaloff>:/task.png</normaloff>:/task.png</iconset>
   </property>
   <property name="text">
    <string>任务攻略</string>
   </property>
  </action>
  <action name="actionDed">
   <property name="icon">
    <iconset>
     <normaloff>:/corp/1000120.png</normaloff>:/corp/1000120.png</iconset>
   </property>
   <property name="text">
    <string>死亡攻略</string>
   </property>
  </action>
  <action name="actionArc">
   <property name="icon">
    <iconset>
     <normaloff>:/corp/1000064.png</normaloff>:/corp/1000064.png</iconset>
   </property>
   <property name="text">
    <string>考古攻略</string>
   </property>
  </action>
  <action name="actionAbyss">
   <property name="icon">
    <iconset>
     <normaloff>:/corp/1000151.png</normaloff>:/corp/1000151.png</iconset>
   </property>
   <property name="text">
    <string>深渊攻略</string>
   </property>
  </action>
  <action name="actionEden">
   <property name="icon">
    <iconset>
     <normaloff>:/corp/1000002.png</normaloff>:/corp/1000002.png</iconset>
   </property>
   <property name="text">
    <string>新伊甸编年史</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="icon">
    <iconset>
     <normaloff>:/about.png</normaloff>:/about.png</iconset>
   </property>
   <property name="text">
    <string>关于我们</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Images/img.qrc"/>
 </resources>
 <connections/>
</ui>
